package.json

```diff
@@ -12,21 +12,33 @@
     "packages/*"
   ],
   "scripts": {
-    "start": "lerna run dev --parallel",
-    "dev": "lerna run dev --parallel",
+    "build": "cd packages/autocannon-ui-frontend && npm run build",
```

Comment:
why not just use lerna to run this command, so it's consistent with all the other commands?

Reply:
We managed to remove lerna